### PR TITLE
Feature/contributing guidelines

### PR DESCRIPTION
**Pull request by hjoaquim**
Updating the contributing guidelines!

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -201,23 +256,26 @@ def test_display_historical(mocker):
 ```
 
 Important notes to go through:
-1. Use `@pytest.mark.record_stdout`to capture `sdtout`.
+
+1. Use `@pytest.mark.record_stdout` to capture `sdtout`.
    1. Recorded stdout is saved as fixtures in a txt file
    2. If your view function only plots a chart, and does not output anything in `sdtout`, you can skip unit testing your `view.py`
-   3. Make sure to mock `visualize_output` method.
+   3. Make sure to mock `visualize_output` whenever possible.
```
Comment (hjoaquim) [inline tests/README.md:263]: @tehcoderer @montezdesousa does this still applies? We still have some references to `visualize_output` across our codebase.

Comment (tehcoderer) [reply tests/README.md:263]: Only if it's a matplotlib chart aka 'portfolio/po'

'OpenBBFigure' has logic not to open pywry window in tests

**Comment by deeleeramone** [comment]
This reference needs to be updated for the new path and file name:

Screenshot 2023-04-10 at 8 25 19 AM

Screenshot 2023-04-10 at 8 25 06 AM

**Comment by deeleeramone** [comment]
The function for `source = BusinessInsider` does not follow the naming convention described in the View section. 

Screenshot 2023-04-10 at 8 38 26 AM

**Comment by deeleeramone** [comment]
In this section it references DarkPoolShorts controller, but that is not the correct one for the example.

Screenshot 2023-04-10 at 8 41 44 AM

**Comment by deeleeramone** [comment]
The example switches again when it is being added to the SDK.  Not following the same example makes it unnecessarily confusing.

Screenshot 2023-04-10 at 8 44 57 AM

**Comment by deeleeramone** [comment]
We should include details on creating a new branch, before the instructions to push.

Screenshot 2023-04-10 at 9 20 20 AM

**Comment by deeleeramone** [comment]
Due Diligence does not exist anymore, references should be updated here.

Screenshot 2023-04-10 at 10 12 37 AM

**Comment by deeleeramone** [comment]
The section on Exporting needs to include SHEET_NAME, and probably the Base Classes section as well. 

Screenshot 2023-04-10 at 6 57 31 PM

**Comment by deeleeramone** [comment]
When we get to the Queue and Pipeline section, followed by the Auto Completer, it would be nice to circle back to the same example so that we can have closure on a complete process. 

Screenshot 2023-04-10 at 7 06 21 PM

**Comment by hjoaquim** [comment]
@deeleeramone thanks a lot for the careful review. I think I was able to address every comment!

**Comment by hjoaquim** [comment]
^ the above commits (https://github.com/OpenBB-finance/OpenBBTerminal/pull/4726/commits/1c7d35fec3ffc8a6e25e26821bb94a6cc60a87cf and https://github.com/OpenBB-finance/OpenBBTerminal/pull/4726/commits/d4fc65791d6c5fc14f03e55bf26b04f76dec25d9) are fixes to the markdown linting and not related with the dashboards - somehow I've messed the commit message, I'm sorry.

**Comment by hjoaquim** [comment]
@Chavithra @jmaslek all good on my side! I think we can merge now

**Pull request merged**